HomeTask2/basicOp.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * evaluates the expression formed with binary operator and numbers
+ * @param {string} operation Any character from , '+','-','*','/'
+ * @param {number} value1 number 1
+ * @param {number} value2 number 2
+ * @example basicOp('+', 4, 7)
+ * 
+ * @returns {number} evaluation of the expression
+ */
+
+function basicOp(operation, value1, value2) {
+    const operators = {
+        '+': (number1, number2) => {
+            return number1 + number2;
+        },
+        '-': (number1, number2) => {
+            return number1 - number2;
+        },
+        '/': (number1, number2) => {
+            return number1 / number2;
```

Comment:
Implement an addition check If number2 is zero.

Reply:
check for number2 is 0 is added